### DISCUSSION TITLE
Update material icons reference URL

### DISCUSSION THREAD

**Discussion by craigreyenga** [requarks/wiki#7129] [Error / Bug Report]
The current URL for the material design icon reference is: https://materialdesignicons.com/

This redirects to a new URL, and presents an alert indicating that it has moved.

The updated URL is https://pictogrammers.com/library/mdi/

**Comment by BPowell76** [comment]
The library needs to also be updated. The project is using version 5.8.x and the now deprecated webfont. I may try to see how to update it, but I'm a noob web developer.
